.github/workflows/ci.yml

```diff
@@ -183,6 +183,21 @@ jobs:
 
         sudo MKOSI_TEST_DEFAULT_VERB=boot python3 -m pytest -m integration -sv tests
 
+    # A single test for now should be sufficient
+    - name: Build/Boot ${{ matrix.distro }}/${{ matrix.format }} UsrOnly
+      if: ${{ matrix.distro == 'ubuntu' && matrix.format == 'gpt_ext4' }}
+      run: |
+        mkdir -p mkosi.default.d
+
+        tee mkosi.default.d/mkosi.default <<- EOF
+        [Distribution]
+        Distribution=${{ matrix.distro }}
+
+        [Output]
+        Format=${{ matrix.format }}
```

Comment:
This is already configured above, no need to repeat it here

Reply:
It's the `UsrOnly=True` option that is important (line 198 currently). I can use `tee --append` but that is potentially more confusing.

Reply:
Now it makes more sense.